tests/test_plugin_management/test_parse_plugin_changes.py

```diff
@@ -0,0 +1,53 @@
+import shutil
+import sys
+import tempfile
+from pathlib import Path
+
+from brainscore_core.plugin_management.parse_plugin_changes import *
```

Comment:
```suggestion
from brainscore_core.plugin_management.parse_plugin_changes import get_all_changed_files, separate_plugin_files, get_plugin_paths, get_plugin_ids
```
star import always bad